### ISSUE TITLE
Carry out research to guide design process -Design

### ISSUE THREAD

**Issue by eloketobe** [zuri-training/anima_lib_team98#10]
To be supervised by @tomietta
[Link to user research carried out](https://www.figma.com/file/0eZpb4IgW6PkyMjjsxujKj/Discovery-%26-Define?node-id=0%3A1)

**Comment by TomiEtta** [comment]
The purpose of the user research was to put our design project into context. It helped us understand the problem we are trying to solve; it told us who our users are, in what context they'll be using our product, and ultimately, what they need from us, the designers.

Here's a snippet of what we found out:

The problem: 
Animation sites are usually complex, contain unnecessary animations and lack ease of use for beginner developers. Usually, these sites are made by only other developers so the user-friendly factor isn’t usually considered.

Solution:
What we have created is an animation library that is easy to understand and offers a variety of basic, necessary animations for even the least tech savvy person. User-friendliness was a top priority.

In conclusion, we expanded on the project brief, user persona, user story, competitive analysis, information architecture and user flow.

Here is the list of people that worked on this task.

@TomiEtta worked as the supervisor, expanded the project brief, made the user story and made reviews and corrections on everything else.
@debbyCokes and @OlamiLekanPhilo worked on the User Flow
@Voiced-debug worked the Information Architecture
@VicRoosevelt worked on the Competitor's Analysis

[This is the link to the User Research.](https://www.figma.com/file/mnP49B5GDsVrVBPIp1Q6g0/Animax-Project-Design?node-id=821%3A5355)

**Comment by eloketobe** [comment]
you dey always finish work fr. 2 chilled zobos for you.
Welldone

**Comment by VicRoosevelt** [comment]
Thanks Tomi. Good work Team

**Comment by Voiced-debug** [comment]
I created the information architecture to create more clearer path for user experience.

[Information architecture](https://www.figma.com/file/mnP49B5GDsVrVBPIp1Q6g0/Animax-Project-Design?node-id=920%3A2622)

**Comment by debbyCokes** [comment]
Worked alongside other team members in carrying out the research for our project 

[Link to the user research](https://www.figma.com/file/mnP49B5GDsVrVBPIp1Q6g0/Animax-Project-Design?node-id=821%3A5355)
